.bazelci/presubmit.yml

```diff
@@ -12,3 +9,11 @@ tasks:
   windows:
     build_targets:
     - "//..."
+  ubuntu2004:
+    # No need to check bzlmod on every platform. This repository only holds
```

Comment:
Can you pin the Bazel version to last_green and use `--enable_bzlmod` instead?

Reply:
Done